tracing/tracing.go

```diff
@@ -0,0 +1,164 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"github.com/theplant/appkit/contexts"
+	"github.com/theplant/appkit/log"
+	"github.com/theplant/appkit/server"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+)
+
+func trace(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		l := log.ForceContext(ctx).With(
+			"context", "appkit/tracing.trace",
+		)
+
+		// Extract tracing propagarion info from HTTP request
```

Comment:
the same typo?
propagarion -> propagation

Reply:
Fixes in 6a0681bd0a86cf9028b6129f34441cda1ac84ff2.